nats/src/main.rs

```diff
@@ -32,19 +33,24 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     Ok(())
 }
 
+type NatsTopic = String;
```

Comment:
I'm not a huge fan of aliasing these for purely internal usage, but don't think I want to block on it. @brooksmtownsend any thoughts?

Reply:
Sorry I missed this one, I like the association with types but personally don't use this pattern